tests/apps/core/test_templates_base.py

```diff
@@ -0,0 +1,52 @@
+"""
+Test suite for the Open Graph for course pages
+"""
+import re
+
+from django.test.utils import override_settings
+
+from cms.test_utils.testcases import CMSTestCase
+
+from richie.apps.core.helpers import create_i18n_page
+
+
+class TemplatesBaseCMSTestCase(CMSTestCase):
+    """Testing the base.html template"""
+
+    def test_templates_base_meta_og_image(self):
+        """
+        Test if the homepage has the default og:image logo
+        """
+        homepage = create_i18n_page("my title", is_homepage=True)
+        homepage.publish("en")
+        url = homepage.get_absolute_url(language="en")
+        response = self.client.get(url)
+        response_content = response.content.decode("UTF-8")
+
+        match = re.search("<meta[^>]+og:image[^>]+(/)?>", response_content)
+        html_meta_og_image = match.group(0)
+        self.assertIn(
+            'content="http://testserver/static/richie/images/logo.png',
+            html_meta_og_image,
+        )
+        self.assertIn("richie/images/logo.png", html_meta_og_image)
+
+    @override_settings(STATIC_URL="https://xyz.cloudfront.net/static/")
+    def test_templates_base_meta_og_image_with_cdn(self):
```

Comment:
I think this test should be in the courses app and in the existing file `courses/test_templates_course_detail.py`

Reply:
Actually it seems redundant with your tests below: https://github.com/openfun/richie/pull/1406/files#diff-5bef639c5eb529ca347f700081792cea0d3530d86d1b385b235f25af1f0cf35bR115

If I did not miss anything it could then be removed?

Reply:
The test documentation was incorrect, but the code was ok. We need to test the og:image value for the base templates specifically the homepage. The tests are similar on test_templates_course_detail_opengraph.py and test_templates_base.py but the first tests on course pages and the second currently is testing on homepage, both have different implementations, so both need different tests.